static/styles/lib/loggedin.scss

```diff
@@ -721,6 +721,48 @@ footer {
     }
 }
 
+.alert-button {
+    visibility: hidden;
+
+    .btn i {
+        color: $primaryColor;
+    }
+
+    .dropdown-menu {
+        width: 400px;
+    }
+
+    @media (max-width: 450px) {
+        .dropdown-menu {
+            width: 70vw;
+            left: -70px !important;
+        }
+        
+        .dropdown-menu::before,
+        .dropdown-menu::after {
+            content: none !important;
+        }
+    }
+
+    .alert-item {
+        padding: 10px 15px;
+        font-size: 14px;
+
+        .alert-title {
+            font-size: 16px;
+        }
+
+        .alert-date,
+        .alert-link {
+            float: right;
+        }
+
+        & + .alert-item {
+            border-top: 1px solid #e5e5e5;
```

Comment:
instead in html code better use classes first and last this is not working everywhere i think...